tests/test_statsd_decorators.py

```diff
@@ -22,7 +22,7 @@ def test_function():
 
 
 def test_should_call_statsd_catch(app_with_statsd, mocker):
-    class CustomException(BaseException):
+    class CustomException(Exception):
```

Comment:
I'm probably missing a lot of context. Why was it necessary the change of which class to inherit

Reply:
Because we are raising this as an exception (`raise CustomException` below), mypy expects that the base class should be `Exception`.
 
![Screen Shot 2021-07-22 at 2 42 23 PM](https://user-images.githubusercontent.com/5498428/126692053-df2c84a0-535d-4954-9f76-6b714cf60bc4.png)

Reply:
It doesn't matter either way but if BaseException is inheriting Exception that should be fine no ?

Reply:
Despite the name, `Exception` was not the base class for `BaseException` 😄